src/con_duct/__main__.py

```diff
@@ -511,10 +514,89 @@ def dump_json(self) -> str:
 
     @property
     def execution_summary_formatted(self) -> str:
-        human_readable = {
-            k: "unknown" if v is None else v for k, v in self.execution_summary.items()
-        }
-        return self.summary_format.format_map(human_readable)
+        formatter = Formatter()
+        return formatter.format(self.summary_format, **self.execution_summary)
+
+
+class Formatter(string.Formatter):
+    # TODO: we might want to just ignore and force utf8 while explicitly .encode()'ing output!
+    # unicode versions which look better but which blow during tests etc
+    # Those might be reset by the constructor
+    OK = "OK"  # u"✓"
+    NOK = "X"  # u"✗"
+    NONE = "-"  # u"✗"
+
+    def __init__(self, *args, **kwargs):
+        super(Formatter, self).__init__(*args, **kwargs)
+        if sys.stdout.encoding is None:
+            lgr.debug("encoding not set, using safe alternatives")
+        elif not sys.stdout.isatty():
+            lgr.debug("stdout is not a tty, using safe alternatives")
+        else:
+            try:
+                "✓".encode(sys.stdout.encoding)
+            except UnicodeEncodeError:
+                lgr.debug(
+                    "encoding %s does not support unicode, " "using safe alternatives",
+                    sys.stdout.encoding,
+                )
+            else:
+                self.OK = "✓"
+                self.NOK = "✗"
+                self.NONE = "✗"
+
+    def convert_field(self, value, conversion):
+        # print("%r->%r" % (value, conversion))
+        if conversion == "D":  # Date
+            if value is not None:
+                return time.strftime("%Y-%m-%d/%H:%M:%S", time.localtime(value))
+            else:
+                return "-"
+        elif conversion == "S":  # Human size
+            # return value
+            if value is not None:
+                # TODO: duplicate functionality with humanize
+                return humanize.naturalsize(value)
```

Comment:
for my reference https://github.com/python-humanize/humanize/blob/main/src/humanize/filesize.py#L12

Reply:
done